Gemfile

```diff
@@ -3,13 +3,17 @@ source 'https://rubygems.org'
 gemspec
 
 group :development do
-  gem 'smart_proxy', git: 'https://github.com/theforeman/smart-proxy',
-                     branch: 'develop'
-  #gem 'smart_proxy', path: '../smart-proxy'
   gem 'pry'
+
+  if ENV.fetch('BUNDLE_SMART_PROXY', '1') == '1'
```

Comment:
I was thinking about `ENV['SMART_PROXY_PATH']` which is the `path` to pass to `gem 'smart_proxy'`

Reply:
oh, RIGHT! :+1: